aws/modules/consolidate-logs/module.ftl

```diff
@@ -0,0 +1,226 @@
+[#ftl]
+
+[@addModule
+    name="consolidatelogs"
+    description="Solution-wide consolidation of logs, intended for consumption by ElasticSearch."
+    provider=AWS_PROVIDER
+    properties=[
+        {
+            "Names" : "namePrefix",
+            "Type" : STRING_TYPE,
+            "Description" : "A prefix appended to component names and deployment units to ensure uniquness",
+            "Default" : "log"
+        },
+        {
+            "Names" : "deploymentProfile",
+            "Type" : STRING_TYPE,
+            "Description" : "The deployment profile id to enable log consolidation on."
+        },
+        {
+            "Names" : "lambdaSourceUrl",
+            "Type" : STRING_TYPE,
+            "Description" : "A URL to the lambda zip package for sending alerts",
+            "Default" : "https://github.com/hamlet-io/lambda-log-processors/releases/download/v1.0.2/cloudwatch-firehose.zip"
+        },
+        {
+            "Names" : "lambdaSourceHash",
+            "Type" : STRING_TYPE,
+            "Description" : "A sha1 hash of the lambda image to validate the correct one",
+            "Default" : "3a6b1ce462aaa203477044cfe83c66f128381434"
+        },
+        {
+            "Names" : "tier",
+            "Type" : STRING_TYPE,
+            "Description" : "The tier to use to host the components",
+            "Default" : "mgmt"
+        }
+    ]
+/]
+
+[#macro aws_module_consolidatelogs
+    namePrefix
+    deploymentProfile
+    lambdaSourceUrl
+    lambdaSourceHash
+    tier]
+
+    [@debug message="Entering Module: consolidate-logs" context=layerActiveData enabled=false /]
+
+    [#local lambdaName = formatName(namePrefix, "lambda")]
+    [#local datafeedName = formatName(namePrefix, "datafeed")]
+
+    [#local product = getActiveLayer(PRODUCT_LAYER_TYPE) ]
+    [#local environment = getActiveLayer(ENVIRONMENT_LAYER_TYPE)]
+    [#local segment = getActiveLayer(SEGMENT_LAYER_TYPE)]
+
+    [#local namespace = formatName(product["Name"], environment["Name"], segment["Name"])]
+    [#local lambdaSettingNamespace = formatName(namespace, lambdaName)]
+
+    [@loadModule
+        settingSets=[
+           {
+                "Type" : "Settings",
+                "Scope" : "Products",
+                "Namespace" : lambdaSettingNamespace,
+                "Settings" : {
+
+                }
+            } 
+        ]
```

Comment:
Shouldn't be required if you aren't injecting any settings

Reply:
Yep cheers - have tidied up.
I had anticipated needing some for the lambda but they weren't necessary.